src/main/resources/pagedump/galen-pagedump.js

```diff
@@ -207,6 +207,24 @@ Suggest.prototype.generateSuggestions = function (items) {
     return "<pre>" + html + "</pre>";
 };
 Suggest.prototype._suggestions = {};
+Suggest.prototype._suggestions.width = function(a, b) {
+           return new Spec(a.name, "width: " + getPercentage(a.area[2], b.area[2]) + "% of " + b.name+"/width");
+   };
+Suggest.prototype._suggestions.height = function(a, b) {
+          return new Spec(a.name, "height: " + getPercentage(a.area[3], b.area[3]) + "% of " + b.name+"/height");
+   };
+Suggest.prototype._suggestions.widthAgainstHeight = function(a, b) {
+    return new Spec(a.name, "width: " + getPercentage(a.area[2], b.area[3]) + "% of " + b.name+"/height");
+};
+Suggest.prototype._suggestions.heightAgainstWidth = function(a, b) {
+    return new Spec(a.name, "height: " + getPercentage(a.area[3], b.area[2]) + "% of " + b.name+"/width");
```

Comment:
Wow, that really is weird :) but lets not make it as a general thing for other users. I would rather have your code for width and height as I think those two are more often used checks.